archetypes/pattern/ideas-for-customization.adoc

```diff
@@ -0,0 +1,15 @@
+---
+title: Ideas for customization
+weight: 30
+---
+
+:toc:
+:imagesdir: /images
+:_content-type: ASSEMBLY
+
+// Add asciidoc modules here
+
+== Next steps
+
+*
+*
```

Comment:
Can we add an archetype for a troubleshooting page as well (`archetypes/pattern/troubleshooting.adoc`)? Like the other pages such as getting started, customization, cluster sizing and so on, troubleshooting is one of the a common pages across most patterns.